main.go

```diff
@@ -104,6 +108,26 @@ func main() {
 		Port:                   9443,
 		HealthProbeBindAddress: ":8081",
 		LeaderElection:         false,
+		NewCache: cache.BuilderWithOptions(cache.Options{
+			SelectorsByObject: cache.SelectorsByObject{
+				// to watch eraser pods
+				&corev1.Pod{}: {
+					Field: fields.OneTermEqualSelector("metadata.namespace", utils.GetNamespace()),
```

Comment:
I _think_ this is still making a cluster scoped watch and having API server filter them down.  You could confirm that one way or the other by reducing the RBAC of the eraser components to use role bindings to properly scope the access.

Reply:
The RBAC is already scoped to the namespace for the manager role, but removing the field to filter by namespace increases the memory again. I think that could be because for the manager we have ClusterRole instead of Role, but we can't change that since we need to watch ImageJobs/ImageLists in default and eraser pods in eraser-system. 

It was the same problem testing with MultiNamespacedCacheBuilder and keeping only the eraser-system and default namespaces, it didn't have an affect on memory consumption.

Reply:
You should create separate cluster roles/roles/cluster roles bindings/role bindings to express the exact permissions of each component.  There is no requirement that everything be done via a single cluster role/cluster role binding.

Reply:
https://youtu.be/Nw1ymxcLIDI is a good resource to understand RBAC fundamentals.

Reply:
thank you this was really helpful! I changed the roles to create a separate Role and ClusterRole